admin-dev/themes/new-theme/js/pages/product/store/actions.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+import * as types from './mutation-types';
+import * as taxCalculations from '../tax-calculations';
+
+/**
+ * Actions responsible for applying domain logic and updating the state after DOM changes.
```

Comment:
Im not sure where to put these comments for whole file when its not a class. Looks abit strange just to float here :smile: doesnt it?

Reply:
It doesn't shock, but I'm not even sure it is required Considering its filename it seems obvious that it is defining the actions of the store for product page